components/CommentModal/CommentModal.js

```diff
@@ -123,9 +196,7 @@ My thoughts:
                 <EmailToMyLegislators />
               </div>
               <div>
-                <links.External href={url}>
-                  Send copy to relevant committee
-                </links.External>
+                <EmailToCommittee />
```

Comment:
What happens if the user doesn't have their legislators specified or the bill doesn't have a committee?

Reply:
if the user hasn't specified their legislators, instead of the "Send copy to your legislatures " there is a link to "Add legislators to your profile to share your testimony"
If the bill doesn't have a committee, then the option to check "send copy to relevant committee" is disabled.